code/components/jomjol_fileserver_ota/server_file.cpp

```diff
@@ -49,8 +49,8 @@ extern "C" {
 
 /* Max size of an individual file. Make sure this
  * value is same as that set in upload_script.html */
-#define MAX_FILE_SIZE   (2000*1024) // 200 KB
-#define MAX_FILE_SIZE_STR "2000KB"
+#define MAX_FILE_SIZE   (8000*1024) // 8 MB
```

Comment:
I would move this in a separate PR!
maybe combine it with https://github.com/jomjol/AI-on-the-edge-device/pull/1073?